packages/lists/i18n/en.i18n.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Lists": {
+    "methods": {
+      "remove": {
+        "lastPublicList": "Cannot delete the last public list."
```

Comment:
Well I think we are running up against a limitation of the all-package approach here perhaps?